Move PRODUCTIONURL to the .config file.

mariacha opened the pull request:
It's an easy way to enable stage_file_proxy in both the test server and your local machine. 

NOTE: on merge, we'll need to update the Deployment documentation here:
https://github.com/opensourcery/opensourcery/wiki/Deployments

mariacha committed:
Move PRODUCTIONURL to the .config file.

jhedstrom commented:
Let's update the README to indicate that the production URL should be edited in `bin/.config`, then I think this is ready to be merged.

mariacha committed:
Adding instructions for production url.

mariacha commented:
@jhedstrom I added a line about the production url to the readme. If we added documentation specifically about deploying to production, that might be a better place to put it, but as it is, that seems as good a place as any.

jhedstrom commented:
@mariacha :+1: I think this can be merged in.
